index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+<head>
+    <script type="text/javascript" src="" async></script>
```

Comment:
Don't add empty script tags, just add the script tag when the script is ready to go.